README.md

```diff
@@ -359,6 +359,44 @@ LLNODE_DEBUG=true TEST_LLNODE_DEBUG=true \
   node test/scan-test.js
 ```
 
+<!--
+### TSC (Technical Steering Committee)
+
+* [bnoordhuis](https://github.com/bnoordhuis) -
+**Ben Noordhuis** &lt;info@bnoordhuis.nl&gt;
+* [cjihrig](https://github.com/cjihrig) -
+**Colin Ihrig** &lt;cjihrig@gmail.com&gt; (he/him)
+* [hhellyer](https://github.com/hhellyer) -
+**Howard Hellyer** &lt;hhellyer@uk.ibm.com&gt;
+* [indutny](https://github.com/indutny) -
+**Fedor Indutny** &lt;fedor.indutny@gmail.com&gt;
+* [joyeecheung](https://github.com/joyeecheung) -
+**Joyee Cheung** &lt;joyeec9h3@gmail.com&gt; (she/her)
+* [mmarchini](https://github.com/mmarchini) -
+**Matheus Marchini** &lt;mmarchini@netflix.com&gt;
+
+### TSC Emeriti
+-->
+
+### Collaborators
+
```

Comment:
Since this is based on a GitHub team you could add markers for [ncu-team](https://github.com/nodejs/node-core-utils/blob/master/docs/ncu-team.md) so that you can use [`ncu-team sync`](https://github.com/nodejs/node-core-utils/blob/master/docs/ncu-team.md#synchronize-files-with-special-blocks) to make it easier to keep the list in sync with the team in the future.

Reply:
Good idea. We could even have a GitHub Action which updates the readme automatically when a team changes.

Reply:
Done